### PR TITLE
Added working path required by wsl

### PR DESCRIPTION
**Pull request by paulli2017**
Kselftest under WSL for both local and remote situation has working path issue that is not implemented in Node class. The current change is to fix this issue and the test passed and Kselftest works as expected under local and azure situation.

### CODE REVIEW

**Comment by paulli2017** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: lisa/node.py [hunk 1]
```diff
@@ -449,6 +449,19 @@ def get_working_path(self) -> PurePath:
         """
         raise NotImplementedError()
 
+    def get_remote_working_path(self) -> PurePath:
```
Comment (squirrelsc) [inline lisa/node.py]: name it like `_get_remote_working_path`, so it's a protected method.

File: lisa/node.py [hunk 2]
```diff
@@ -658,17 +671,7 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
...
-        else:
-            remote_root_path = Path("%TEMP%")
-
-        working_path = remote_root_path.joinpath(
-            constants.PATH_REMOTE_ROOT, constants.RUN_LOGIC_PATH
-        ).as_posix()
-
-        # expand environment variables in path
-        return self.get_pure_path(self.expand_env_path(working_path))
+        return super().get_remote_working_path()
```
Comment (squirrelsc) [inline lisa/node.py]: `self.get_remote_working_path()` is clearer.

File: lisa/node.py (hunk 1 above)
Comment (paulli2017) [reply lisa/node.py]: the change is done, thanks

**Comment by squirrelsc** [comment]
@LiliDeng LGTM

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**